types/jexcel/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+      "export-just-namespace": false
```

Comment:
Could you enable this rule?

Reply:
Thank you for the review.
I'm sorry for the my mistake...
Could you check the src again?